tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -20,6 +20,29 @@ module.exports = {
       await this.openSideBar(fileName)
       return client.page.FilesPageElement.appSideBar().selectAccordionItem('people')
     },
+    /**
+     * @param {string} fileName
+     * @return {Promise<*>}
+     */
+    isPreviewImageDisplayed: async function(fileName) {
+      await this.waitForFileVisible(fileName)
+      const element = util.format(this.elements.previewImage.selector, fileName)
+      let previewStatus = ''
+      await this.useXpath()
+        .waitForElementVisible(element)
+        .getAttribute(
+          { selector: element, locateStrategy: this.elements.previewImage.locateStrategy },
+          'data-preview-loaded',
+          result => {
+            if (result.value === 'true') {
+              previewStatus = true
+            } else if (result.value === 'disabled') {
+              previewStatus = false
+            }
```

Comment:
```suggestion
    previewStatus =  result.value === 'true'
```

I think it is safe to assume that all values other than 'true' means it's not displayed.